roles/logs_client/tasks/deploy.yml

```diff
@@ -1,13 +1,26 @@
 ---
+- name: Install gnutls and newer openssl
+  ansible.builtin.yum:
+    name:
+      - gnutls
+      - gnutls-utils
+      - openssl11
+      - openssl11-devel
```

Comment:
Do you need both for rsyslog? That might get confusing. AFAIK they are alternatives. You use either one or the other, but not both for the same software.

Reply:
Yes, client needs both.
Initially we wrote the entire playbook in what was (via available documentation) recommended = `gnutls`.
But just last day I switched all the communication to the recommended `openssl` (to semi-quote the documentation "... by default we have set gnutls, but openssl is recommended"). The creation of CA, certificate request creation and the signing with the CA is still done with gnutls and from what I understand it should not matter what tool generates CA and certificates.